release/cli/pkg/bundles/bottlerocket.go

```diff
@@ -61,23 +67,30 @@ func bottlerocketDefaultArtifact(r *releasetypes.ReleaseConfig, metadataFile, im
 	}, nil
 }
 
-func bottlerocketKubeadmBootstrapArtifact(r *releasetypes.ReleaseConfig, eksDReleaseChannel string, imageDigests map[string]string) anywherev1alpha1.Image {
-	artifacts := r.BundleArtifactsTable[fmt.Sprintf("bottlerocket-bootstrap-%s", eksDReleaseChannel)]
+func bottlerocketKubeadmBootstrapArtifact(r *releasetypes.ReleaseConfig, eksDReleaseChannel string, imageDigests sync.Map) (anywherev1alpha1.Image, error) {
```

Comment:
Same comment here (and the others as well) about passing an interface instead of `sync.Map`